core/modules/utils/dom/color.js/tiddlywiki.files

```diff
@@ -0,0 +1,21 @@
+{
+	"tiddlers": [
+		{
+			"file": "color.global.0.5.2.js",
+			"fields": {
+				"type": "application/javascript",
+				"title": "$:/core/modules/utils/dom/color.js",
+				"module-type": "library"
```

Comment:
IMO we should add 2 more fields: `version: 0.5.2` and `repo: https://github.com/color-js/color.js` to make it easier for us to maintain

Reply:
Hi @pmario if we were going to add those fields then we should do so consistently for all the 3rd party libraries in the core.

Reply:
I think that would be a good idea and we have to start somewhere.